src/Nordea/Components/InfoPanel.elm

```diff
@@ -0,0 +1,47 @@
+module Nordea.Components.InfoPanel exposing (..)
+
+import Css exposing (int, pseudoClass, rem)
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes as Attributes
+import Nordea.Resources.Colors as Colors
+
+
+type alias InfoPanelContent =
+    { label : String
+    , info : String
+    }
+
+
+view : List InfoPanelContent -> Html msg
+view panelInfoList =
+    Html.div
+        [ Attributes.css
+            [ Css.backgroundColor Colors.grayCool
+            , Css.width (rem 25)
+            , Css.boxSizing Css.borderBox
+            , Css.borderRadius (rem 0.5)
+            , Css.border3 (rem 0.125) Css.solid Colors.grayMedium
+            ]
+        ]
+        (panelInfoList |> List.map (\panelInfo -> viewLabelInfoBlock panelInfo))
+
+
+viewLabelInfoBlock : InfoPanelContent -> Html msg
+viewLabelInfoBlock infoPanelContent =
+    Html.div
+        [ Attributes.css
+            [ Css.padding2 (rem 0.75) (rem 0.5)
+            , pseudoClass "not(:last-child)" [ Css.borderBottom3 (rem 0.125) Css.solid Colors.grayMedium ]
+            , Css.firstChild [ Css.fontWeight (int 500), Css.fontSize (rem 1.125) ]
+            ]
+        ]
+        [ Html.div
+            [ Attributes.css
+                [ Css.fontSize (rem 0.875)
+                , Css.fontWeight (int 200)
+                , Css.marginBottom (rem 0.25)
+                ]
+            ]
+            [ Html.text infoPanelContent.label ]
+        , Html.text infoPanelContent.info
```

Comment:
Can the Text component be used here?

Reply:
It might be used for the label? But for the info text it will change both font size and weight, so I dont know if that will just complicate the work? 🤷

Reply:
Hmm, I think we have a "bold" variant of each text, but is the font size something custom?

Reply:
For the first information listed, the font size should be bigger and also bold. For the others its just 1 rem. and the font weight should also be regular. Since there is more than one adjustment, I went for regular text instead of trying to make our text components fit this solution.

But I am open for suggestions! 😊

Reply:
I think this approach is fine, but maybe specify the Nordea font explicitly? Just in case the users of the component havent defined it globally.